app/controllers/api/messages_controller.rb

```diff
@@ -1,17 +1,36 @@
 class Api::MessagesController < ApplicationController
-  before_action :authenticate_request
+  before_action :authenticate_request, :validate_user
 
   def send_message
-    # TODO: What happens if the Chatroom can not be found?
-    @chatroom = Chatroom.find(params[:id])
```

Comment:
No need to use an instance variable here, chatroom (without @) should be necessary